deployments/charts/kuma/values.yaml

```diff
@@ -2,11 +2,12 @@
 global:
   namespace: "kuma-system"
   image:
+    # -- Default registry for all Kuma Images
     registry: "kong-docker-kuma-docker.bintray.io"
     # Defaults to .Chart.AppVersion
     # tag: ""
 
-# Whether or not to patch the target namespace with the system label
+# --- Whether or not to patch the target namespace with the system label
```

Comment:
Sometimes it's `--` (two dashes) other times it's `---`(three dashes). Is that ok?

Reply:
oops. it's `--` only need to do some cleanup.